src/datetime/mod.rs

```diff
@@ -79,11 +79,141 @@ pub enum SecondsFormat {
     __NonExhaustive,
 }
 
-/// ISO 8601 combined date and time with time zone.
+/// `DateTime` is the preferred type for working with real-world timestamps. It contains all the
```

Comment:
For best presentation in rustdoc, the first line should be a high-level, self-contained description of the type. I usually prefer not to end that line with a period, since it's usually more like a sentence fragment than a full sentence. Add an empty line after that, then start paragraphs of more detailed documentation.